go.mod

```diff
@@ -20,8 +20,8 @@ replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.2.13
 	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20180202092358-40e2722dffea
 	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190104202606-0ac367fd6bee+incompatible
-	github.com/florianl/go-conntrack => github.com/florianl/go-conntrack v0.1.1-0.20191002182014-06743d3a59db
-	github.com/iovisor/gobpf => github.com/DataDog/gobpf v0.0.0-20200728095841-88e65b29b186
+	github.com/florianl/go-conntrack => github.com/florianl/go-conntrack v0.2.0
+	github.com/iovisor/gobpf/elf => github.com/DataDog/gobpf/elf v0.0.0-20200902205915-58bf266929a4
```

Comment:
I think you need to remove the `elf` postfix, I can't build `system-probe` with it.